eva_submission/eload_ingestion.py

```diff
@@ -212,6 +221,31 @@ def load_from_ena(self):
             self.eload_cfg.set(self.config_section, 'ena_load', value='failure')
             raise e
 
+    def load_from_ena_from_analysis(self, analysis_accession):
```

Comment:
Can this be a single `run_load_from_ena` method, with the `-A` parameter added whenever `analysis_accession` is passed?  Just to reduce repeated code.

Reply:
Done.